src/_pytest/config/__init__.py

```diff
@@ -1305,6 +1316,69 @@ def _getconftest_pathlist(
             values.append(relroot)
         return values
 
+    def _get_append_ini_values(self, name: str, ini_type) -> List[str]:
+        value = []
+        # append_ini is a list of "ini=value" options
+        # append all values if multiple values are set for same ini-name,
+        # e.g. -a foo=bar1 -a foo=bar2 will append 'bar1 bar2' to foo
+        for ini_config in self._append_ini:
+            try:
+                key, append_ini_value = ini_config.split("=", 1)
+            except ValueError:
+                raise UsageError(
+                    "-a/--append-ini expects option=value style (got: {!r}).".format(
+                        ini_config
+                    )
+                )
+            else:
```

Comment:
I think the `else` is not needed here, would remove it to reduce the indentation a bit (easier to read).

Reply:
I agree
This was existing code that was moved around. Let me remove it